spec/builtin/web/dir_enum_spec.rb

```diff
@@ -57,19 +57,19 @@ class App < Sinatra::Base
       set :port, 80
 
       get '/' do
-        halt 200
+        [200, ['200']]
       end
 
       get '/admin' do
-        halt 200
+        [200, ['200']]
       end
 
       get '/downloads' do
-        halt 200
+        [200, ['200']]
       end
 
       get '/secret' do
-        halt 200
+        [200, ['200']]
```

Comment:
Use something like "Hello world" here or "Body for /downloads", so it doesn't accidentally get confused with the status code.